components/ArticlesList.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useState } from 'react';
+import ArticleCard from './ArticleCard';
+import { fetchArticles } from '../apis';
+
+const ArticlesList = () => {
+  const [articles, setArticles] = useState([]);
+  useEffect(() => {
+    fetchArticles()
+    .then(({articles}) => {
+      setArticles(articles);
+    })
+  })
+
```

Comment:
Leaving out a dependancy array can be dangerous! Please do add it in even if it is empty

Reply:
Thanks Rayhaan. Added that in